scripts/lint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex # fail on first error, print commands
+
+SRC_DIR=${SRC_DIR:-$(pwd)}
+
+echo "Checking code style with pylint..."
+python3 -m pylint bambi/
```

Comment:
I'd just as well use `python` instead of `python3` given python2 EOL is so soon -- I don't think this matters, though...

Reply:
2ddb42e